### DISCUSSION TITLE
Implement Bankcontact gateway type for Stripe

### DISCUSSION THREAD

**Discussion by stenvdb** [craftcms/commerce-stripe#168] [Ideas]
Bankcontact is the most popular payment method in Belgium, would be very nice to see it get implemented. Created this separate issue as per request https://github.com/craftcms/commerce-stripe/issues/28#issuecomment-487504831

https://stripe.com/docs/sources/bancontact

**Comment by andris-sevcenko** [comment]
As we gear up for [EU SCA regulation](https://stripe.com/guides/strong-customer-authentication) coming in effect on Sep 14th, 2019, we have to assess where we stand on things. The SCA regulations would make implementing this in the current state obsolete in September. 

On our end, it means that implementation of this depends [on the entry for Bankcontact](https://stripe.com/docs/payments/payment-methods#transitioning) changing from "Planned" to "Supported".

It shouldn't take long, because that's a bunch of customers that Stripe doesn't want to lose on September, either.

**Comment by stenvdb** [comment]
Thanks for the update! We can always go with the mollie plugin for bankcontact support for now. I'll follow up on this as well.
